### ISSUE TITLE
Support modal dialogs

### ISSUE THREAD

**Issue by TiGR** [jpedroribeiro/Baseliner#74]
If there is an open modal dialog on the page, baseline is added under that dialog, so, you can't use it on modals.

Maybe it should track if there are open modals on the page and if there are, it would add the baseline not to body, but to topmost modal dialog (topmost layer).

**Comment by jpedroribeiro** [comment]
Interesting scenario, difficult to pinpoint which element is a modal unless you're talking about an actual `<dialog>`  HTML element.
Thanks for the feedback, I'll see what I can come up with.

**Comment by TiGR** [comment]
I'm talking about actual dialog HTML elements.

**Comment by jpedroribeiro** [comment]
Tested around on some page that use `<dialog>` and it seems to work fine (as in, Baseliner is on top of all layers), my suspicion is that in your case it could be a `z-index` clash. 
Could you provide the URL of where you saw this issue?

**Comment by TiGR** [comment]
Have you opened it via `showModal()`? Also, services like codepen do not work, since they all use iframes, which put them into separate stack and you can't reproduce it.

You can run this code in inspector on any site to reproduce it:

```javascript
const dialog = document.createElement('dialog');
document.documentElement.appendChild(dialog);
dialog.showModal();
```

You can use escape to close the modal.

**Comment by jpedroribeiro** [comment]
I have just published [Baseliner v7.2](https://chromewebstore.google.com/detail/baseliner/agoopbiflnjadjfbhimhlmcbgmdgldld?hl=en-GB) with an update for this. It should target `dialog` tags that are programatically added to the DOM. 
Let me know how it goes for you.

**Comment by TiGR** [comment]
Perfect! Thank you, everything now works as expected.
